tikv/region.go

```diff
@@ -98,6 +98,9 @@ var NewCodecPDClient = locate.NewCodecPDClient
 // NewCodecPDClientWithKeyspace creates a CodecPDClient in API v2 with keyspace name.
 var NewCodecPDClientWithKeyspace = locate.NewCodecPDClientWithKeyspace
```

Comment:
should probably remove this and it's reference in test/tidb since we are moving to NewCodecPDClientWithKeyspaceMeta

Reply:
Won't clean up NewCodecPDClientWithKeyspace in this PR.